bika/lims/content/bikasetup.py

```diff
@@ -757,6 +757,45 @@ class PrefixesField(RecordsField):
                           "will be displayed too. By default, disabled.")
         )
     ),
+    BooleanField(
+        'SamplingBarEnabledAnalysisRequests',
+        schemata="Analyses",
+        default=False,
+        widget=BooleanWidget(
+            label=_("Enable the sampling bar in Analysis Requests lists"),
```

Comment:
Shouldn't be "Display an advanced filter bar in Analysis Requests lists"? Ooops, I see the Field name is "SamplingBarEnabledAnalysisRequests". Maybe I'm missing something